README.md

```diff
@@ -1 +1,161 @@
-# audit-service-deploy
\ No newline at end of file
+## Audit Service Deployment
+
+### Introduction
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  This definition is registered with the Audit service and used to create a Java SDK to record the audited events.  The Audit service is multi-tenant aware.  Applications send events using the generated Java SDK to Elasticsearch where they are indexed according to each tenant.
+
+### Deployment Repository
+
+This repository provides the necessary files to easily get started using the Audit Service.
+
+The only pre-requisite required to get started is that Docker must be available on the system.
+
+The deployment files are in Docker Compose v3 format, and they are compatible with both Docker Compose and Docker Stack.
+
+The deployment file references at present only references Elasticsearch.
```

Comment:
"The deployment file references at present only references Elasticsearch." makes no sense to me.